testing/src/test/java/org/motechproject/nms/testing/it/api/OpsControllerBundleIT.java

```diff
@@ -170,11 +170,19 @@ public void testBadContactNumberAddFlwRequest() throws IOException, InterruptedE
     // Create valid new flw
     @Test
     public void testCreateNewFlw() throws IOException, InterruptedException {
-        AddFlwRequest addFlwRequest = getAddRequest();
+        AddFlwRequest addFlwRequest = getAddRequestASHA();
         HttpPost httpRequest = RequestBuilder.createPostRequest(addFlwEndpoint, addFlwRequest);
         assertTrue(SimpleHttpClient.execHttpRequest(httpRequest, HttpStatus.SC_OK, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD));
     }
 
+    // Validating whether the type is ASHA or not
+    @Test
```

Comment:
Is there a negative test to confirm that this fails if the designation is wrong?